### PR TITLE
Updated several RVV instructions

### LINKED ISSUE #69: Missing Vector Integer Add-with-Carry / Subtract-with-Borrow Instructions Opcodes
The following instructions mentioned in spec:

## 11.4. Vector Integer Add-with-Carry / Subtract-with-Borrow Instructions

But missing in `opcodes-rvv` file

```
vmadc.vv
vmadc.vx
vmadc.vi

vmsbc.vv
vmsbc.vx
```

### LINKED ISSUE #70: Missing 11.8. Vector Integer Comparison Instructions
The following instructions mentioned in spec:

## 11.8. Vector Integer Comparison Instructions

But missing in `opcodes-rvv` file

```
vmsgeu.vx 
vmsge.vx
```

### LINKED ISSUE #71: Missing 14. Vector Reduction Operations
The following instructions mentioned in spec:

## 14. Vector Reduction Operations

But missing in `opcodes-rvv` file

```
vfredusum.vs
vfwredusum.vs
```

### LINKED ISSUE #72: Missing 15. Vector Mask Instructions
The following instructions mentioned in spec:

## 15. Vector Mask Instructions

But missing in `opcodes-rvv` file

```
vmmv.m
vmclr.m
vmset.m
vmnot.m

vcpop.m
```

### LINKED ISSUE #73: Name change vle1.v, vse1.v --> vlm.v, vsm.v
New spec in chaper:

## 7.4. Vector Unit-Stride Instructions

Has new names for some instructions:

```
vlm.v
vsm.v
```

Old names in `opcodes-rvv` file

```
vle1.v
vse1.v
```

### PR DESCRIPTION
**Pull request by 2henwei**
Close #69, close #70, close #71, close #72, close #73

Some missing RVV instructions are pseudo-instructions for assembler to handle. I don't think they have to be added into the parser.

### CODE REVIEW

File: opcodes-rvv [hunk 1]
```diff
@@ -305,9 +307,11 @@ vrgather.vv     31..26=0x0c vm vs2 vs1 14..12=0x0 vd 6..0=0x57
 vrgatherei16.vv 31..26=0x0e vm vs2 vs1 14..12=0x0 vd 6..0=0x57
 
 vadc.vvm       31..26=0x10 25=0 vs2 vs1 14..12=0x0 vd 6..0=0x57
-vmadc.vvm      31..26=0x11 vm   vs2 vs1 14..12=0x0 vd 6..0=0x57
+vmadc.vvm      31..26=0x11 25=0 vs2 vs1 14..12=0x0 vd 6..0=0x57
+vmadc.vv       31..26=0x11 25=1 vs2 vs1 14..12=0x0 vd 6..0=0x57
```
Comment (aswaterman) [inline opcodes-rvv:311]: This looks wrong.  There is no instruction named `vadc.vv` in the spec.

Comment (2henwei) [reply opcodes-rvv:311]: That is vmadc.vv

Comment (aswaterman) [reply opcodes-rvv:311]: Yes, I made a typo, but there's also no `vmadc.vv` in the spec.

**Comment by aswaterman** [comment]
Going forward, please make these changes as separate commits.  It's hard to review and verify the changes when they're made in bulk.

**Review by aswaterman — APPROVED** [review]

File: opcodes-rvv (hunk 1 above)
Comment (2henwei) [reply opcodes-rvv:311]: Hm, I saw them in section 11.4
```
 # vd.mask[i] = carry_out(vs2[i] + vs1[i])
 vmadc.vv    vd, vs2, vs1      # Vector-vector, no carry-in

 # vd.mask[i] = carry_out(vs2[i] + x[rs1])
 vmadc.vx    vd, vs2, rs1      # Vector-scalar, no carry-in

 # vd.mask[i] = carry_out(vs2[i] + imm)
 vmadc.vi    vd, vs2, imm      # Vector-immediate, no carry-in
 ```

Comment (2henwei) [reply opcodes-rvv:311]: Okay cool!

**Comment by 2henwei** [comment]
Ah I'm just working on separating commits. I'll notice it next time. 🙂

**Pull request merged**